sum.c

```diff
@@ -0,0 +1,38 @@
+//Matthew Kellerman - code prints out culmative sum of array's elements
+#include <stdio.h>
```

Comment:
Really nice job! Good comments, nice take on the assignment (handling array size inputs of < 1 was 💯)